plugin-collab-note/src/init/open-collab-note.commad.ts

```diff
@@ -8,7 +8,7 @@ import {
 const opener = require('opener');
 
 function isValidUrl(url: string): boolean {
-  const urlRegex = /^((http(s)?):\/\/)([wW]{3}\.)?[a-zA-Z0-9\-.]+\.[a-zA-Z]{2,}(\.[a-zA-Z]{2,})?$/g;
+  const urlRegex = /^((http(s)?):\/\/)([wW]{3}\.)?[a-zA-Z0-9\-.]+\.[a-zA-Z]{2,}(\.[a-zA-Z]{2,})?(\/)?$/g;
```

Comment:
maybe you should use some lib to validate url instead? for instance it seems like it won't accept a port number like
`example.com:12345/something`

Reply:
I added a library to validate the URL. but I had to add another ReqExp in conjunction with the validator to bypass the localhost URL.

Although the validator is marking `example.com:12345/something` as valid. The URL opener library isn't opening it in a browser. As it only recognizes the string as URL if the prefix is either `http://` or `https://`

Reply:
From the opener library readme:
> Because Windows has start, Macs have open, and *nix has xdg-open.
so it's the underlying command which takes `example.com:12345/something` as a file location rather than a URL and tries to open it as a file.

I can think of 2 options now:
- Add `http://` prefix as default if there no prefix
- Check for the prefix `http://` or `https://` along with validator

What would you suggest?

Reply:
Yeah, sorry I omitted the prefix, I meant that the port will be a problem for the regex. So my original comment should've read:

> for instance it seems like it won't accept a port number like http://example.com:12345/something

Reply:
I made the changes accordingly, now the plugin alsp shows an error popup for URL not containing prefix of `http://` or `https://`. Is it ready to be merged?